pkg/v1/tkg/constants/clusterclass_mapping.go

```diff
@@ -279,7 +279,8 @@ const (
 
 	SPEC = "spec"
 
-	TopologyClassIncorrectValueErrMsg = "input cluster class file, attribute spec.topology.class has no value or incorrect value or not following correct naming convension"
+	TopologyClassIncorrectValueErrMsg                = "input cluster class file, attribute spec.topology.class has no value or incorrect value or not following correct naming convension"
```

Comment:
misspelling 'convension' should be 'convention'

Reply:
Thanks for catching this. fixed.